x/lockup/types/genesis.go

```diff
@@ -13,8 +11,5 @@ func DefaultGenesis() *GenesisState {
 // Validate performs basic genesis state validation returning an error upon any
 // failure.
 func (gs GenesisState) Validate() error {
-	if gs.LastLockId < 0 {
```

Comment:
Why was this removed?

Reply:
This is a `uint` so it cannot be less than 0